### DISCUSSION TITLE
Use with portrainer

### DISCUSSION THREAD

**Discussion by piyushimself** [projectdiscovery/nuclei#3039] [Q&A]
I am thinking to use nuclei in docker. Can you suggest something so i can use and access it by using portrainer?

**Comment by MetzinAround** [comment]
@piyushimself We do have a docker image for nuclei, so you should be able to use it there. Not sure how to use portrainer myself, but we have the Docker image! 

https://hub.docker.com/r/projectdiscovery/nuclei
